lib/ruby/truffle/mri/bigdecimal/jacobian.rb

```diff
@@ -0,0 +1,87 @@
+#
```

Comment:
We usually just do a require_relative to avoid the copy, see `lib/ruby/truffle/mri/abbrev.rb` for instance.
Or `require_relative '../../../stdlib/rubygems/' + File.basename(__FILE__)` if you do not want to hardcode the basename of the current file.